lib/radiator/podcasts.ex

```diff
@@ -0,0 +1,297 @@
+defmodule Radiator.Podcasts do
```

Comment:
Could Broadcast be a better context name than Podcasts? [Contexts are dedicated modules that expose and group related functionality.](https://hexdocs.pm/phoenix/contexts.html#thinking-about-design)

Reply:
Thanks for the input. I agree, I am not satisfied with the naming of the context `Podcasts` as well. It should differentiate to the new outliner tool, we are working on, the media files a podcast will be associated with. It is the base structure for all phases of the app. Our core. For my feeling `Broadcasts` might be a bit misleading because especially in phase 1 the podcast has not been broadcasted yet. 
Naming is so hard ;)

Reply:
Yeah. :) `Podcasts` is fine as a placeholder then.

Reply:
How about the name **Show**? (Tim suggested that once)
So the hierarchy would be:
1. Podcast.Network (e.g. Metaebene or ARD or BBC)
2. Podcast.Show (e.g. Freakshow or Minkorrekt)
3. Podcast.Episode (e.g. FR123 or Mi014)